src/components/TextPicker/TextSelectorModal.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from 'react';
```

Comment:
What part of the app is this exactly for? feels odd we would not have anything in place already

Reply:
Here we want to open RHP to select the name of the tax rate. I've added it as a new Input Form component, to ease the development. The main reason for this approach is the validation, as it needs to happen on the main screen.

Typically, we don't open an RHP for text values, we just use `TextInput`. So, we might consider using `TextInput` and enabling the user to type the name of the tax in the main screen, instead of opening an RHP.

Reply:
Ok thanks for explaining. I think inline text input might be something we explore in future, but not now